test/user-test.js

```diff
@@ -61,79 +61,134 @@ describe('User', () => {
     expect(user1.name).to.equal("Saige O'Kon");
   });
 
-  it('Should have a property of favoriteRecipes with a default value',
-    function() {
-      expect(user1.favoriteRecipes).to.deep.equal([]);
+  it.skip('Should be able to check ingredients in User/s pantry for a given recipe', () => {
+    expect(user1.checkPantry(recipeIngredients)).to.eql('You have the ingredients!');
+  });
+
+  it.skip('Should inform User if they lack required ingredients for a given recipe', () => {
+    expect(user1.checkPantry(recipeIngredients)).to.eql(missingIngredientsWithPrice);
+  });
```

Comment:
Are these currently passing? Is there a reason they're skipped?

Reply:
They are skipped because they are dependent on methods in the pantry class which we are currently working on.  So I'll make sure to let him know to change these once the pantry class is finished.